hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -46,6 +46,8 @@
   private static final String INSERT_PARALLELISM = "hoodie.insert.shuffle.parallelism";
   private static final String BULKINSERT_PARALLELISM = "hoodie.bulkinsert.shuffle.parallelism";
   private static final String UPSERT_PARALLELISM = "hoodie.upsert.shuffle.parallelism";
+  private static final String INSERT_WRITE_BUFFER_LIMIT = "hoodie.insert.write.buffer.limit";
```

Comment:
I would just call this `hoodie.write.buffer.limit` since we intend to eventually add this to MergeHandle as well. Please rename the variables everywhere accordingly.

Reply:
actually `hoodie.write.buffer.limit.bytes`

Reply:
done. Changed it to "hoodie.write.buffer.limit.bytes".